docs/upgrade-guide.md

```diff
@@ -2,6 +2,10 @@
 
 ## Upgrading to v3.0
 
+**Transpilation**
```

Comment:
Good docs are really appreciated. There are some concerns from folks about how hard this tool is to use.

If anything in the main doc text can also be improved that would be awesome.

Reply:
Sorry, that comment applies to ocular. Perhaps I didn't pay attention to doc updates in that PR.